flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/BlockingCallMonitoringThreadPool.java

```diff
@@ -101,4 +106,12 @@ public boolean isShutdown() {
 	public void shutdownNow() {
 		executor.shutdownNow();
 	}
+
+	public int getMaximumPoolSize() {
```

Comment:
package-private and `@VisibleForTesting`?

Reply:
I don't mind either way. package private sounds good.